### PR TITLE
Adding Color Space documentation 

### LINKED ISSUE #774: Color Space blog post
Blog post and documentation page on using color space in Babylon.js

### PR DESCRIPTION
**Pull request by PatrickRyanMS**
Adding documentation to authoring assets section on how to use color space in Babylon.js. Includes text and images for doc.

closes #774

### CODE REVIEW

File: content/preparingArtForBabylon/controllingColorSpace.md
Comment (RaananW) [inline content/preparingArtForBabylon/controllingColorSpace.md]: it would be better to starts with h2 (`##`) so that the menu will be generated correctly. Unless you did that deliberately?

Comment (RaananW) [reply content/preparingArtForBabylon/controllingColorSpace.md]: Same to all following titles

File: content/preparingArtForBabylon/controllingColorSpace.md
Comment (RaananW) [inline content/preparingArtForBabylon/controllingColorSpace.md:53]: if this is a specific language you can define it here (like : \`\`\`javascript). Otherwise leave it as is :-)

**Review by RaananW — APPROVED** [review]
LGTM! just two comments before merging

File: content/preparingArtForBabylon/controllingColorSpace.md
Comment (PatrickRyanMS) [reply content/preparingArtForBabylon/controllingColorSpace.md]: I am not a fan of the HR that is added for h2, but I can change it.

File: content/preparingArtForBabylon/controllingColorSpace.md
Comment (PatrickRyanMS) [reply content/preparingArtForBabylon/controllingColorSpace.md:53]: it's not a specific language. It's just math that I wanted to stand apart from the rest of the text since it is math

File: content/preparingArtForBabylon/controllingColorSpace.md
Comment (RaananW) [reply content/preparingArtForBabylon/controllingColorSpace.md]: the main importance is to have the menu rendered correctly (it stops at h3). we'll see how we can fix the h2 design.
Feel free to merge this if you find it enough as is

File: pages/documentationPage.module.scss
Comment (RaananW) [inline pages/documentationPage.module.scss]:
```suggestion
        font-size: 1rem;
```

File: pages/documentationPage.module.scss
Comment (RaananW) [inline pages/documentationPage.module.scss]:
```suggestion
        font-size: 1rem;
```

**Pull request merged**
